op-node/rollup/finality/finalizer_test.go

```diff
@@ -190,8 +199,18 @@ func TestEngineQueue_Finalize(t *testing.T) {
 		l1F.ExpectL1BlockRefByNumber(refD.Number, refD, nil)
 		l1F.ExpectL1BlockRefByNumber(refD.Number, refD, nil)
 
+		l2F := &testutils.MockL2Client{}
+		defer l2F.AssertExpectations(t)
+		helperL2BlockRefByNumberWithTimes(t, l2F, 1, refA1, refB0, refB1, refC0, refC1)
```

Comment:
i usually don't see people add "helper" to the name of a helper function. let's name it to `mockL2BlockRefByNumberWithTimes`

Reply:
ok